qtranslate_frontend.php

```diff
@@ -330,7 +330,7 @@ function qtranxf_excludeUntranslatedPosts($where,&$query) {//WP_Query
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: post_type is empty: $query: ',$query, true);
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: $where: ',$where);
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: is_singular(): ',is_singular());
-	$single_post_query=is_singular();
+	$single_post_query=$query->is_singular();
```

Comment:
Is there a specific WP user, whose problem is solved by this change? Could you explain the logic why this is needed?